tests/html/test_html.py

```diff
@@ -0,0 +1,28 @@
+import sys
+
+from pytest_mock import MockerFixture
+
+import pystac
+from pystac.html.jinja_env import get_jinja_env
+from tests.utils import TestCases
+
+
+def test_item_html() -> None:
+    item_stac_href = TestCases.get_path("data-files/item/sample-item.json")
+    item_html_href = TestCases.get_path("data-files/html/sample-item.html")
+    item = pystac.Item.from_file(item_stac_href)
+    item.set_self_href(None)
+    assert item._repr_html_() == open(item_html_href).read()
```

Comment:
I'm a little uneasy about failing tests if the HTML representation changes just a little bit. What do you think about just checking that the `_repr_html_()` produces valid HTML, and nothing more?

Reply:
Yeah, that makes sense. I don't like monolothic comparisons either.

Reply:
I added a simple html parser in place of the html string comparison. Uncovered what appears to be a bug in the `_html_repr()` - see #955.